sale_quotation_number/__manifest__.py

```diff
@@ -7,7 +7,7 @@
 {
     'name': 'Sale Quotation Numeration',
     'summary': "Different sequence for sale quotations",
-    'version': '11.0.1.1.0',
+    'version': '12.0.1.1.0',
     'category': 'Sales Management',
     'website': 'https://odoo-community.org/',
```

Comment:
Website URL should be link this 'https://github.com/OCA/sale-workflow/'

Reply:
Done